x-pack/plugins/cases/public/mocks.ts

```diff
@@ -19,9 +20,13 @@ const createStartContract = (): jest.Mocked<CasesUiStart> => ({
   hooks: {
     getUseCasesAddToNewCaseFlyout: jest.fn(),
     getUseCasesAddToExistingCaseModal: jest.fn(),
+    useCasesToast: jest.fn(),
```

Comment:
I don't think we should export the `useCasesToast` hook.

Reply:
I just pushed a commit for that :)